<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_35">
    <item row="7" column="1" colspan="5">
     <widget class="QPushButton" name="solveButton">
      <property name="text">
       <string>Solve</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="5">
     <widget class="QPushButton" name="ResetLabel">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="5">
     <layout class="QGridLayout" name="sudokuGrid">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </item>
    <item row="6" column="1" colspan="5">
     <widget class="QLabel" name="dialog">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>15</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
        <kerning>false</kerning>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QPushButton" name="PreviousButton">
      <property name="text">
       <string>&lt; Previous</string>
      </property>
     </widget>
    </item>
    <item row="8" column="5">
     <widget class="QPushButton" name="NextButton">
      <property name="text">
       <string>Next &gt;</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QPushButton" name="beginStep">
      <property name="text">
       <string>&lt;&lt; Begin</string>
      </property>
     </widget>
    </item>
    <item row="9" column="5">
     <widget class="QPushButton" name="lastStep">
      <property name="text">
       <string>Last &gt;&gt;</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3" rowspan="2">
     <widget class="QLabel" name="StepLabel">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
